.codecov.yaml

```diff
@@ -18,6 +18,10 @@ comment:
   behavior: default
 ignore:
```

Comment:
another question - what does ignore actually do?

ie. it would be good to know the coverage for these packages and not ignore them.

We only want to ignore these packages when calculating the total coverage across this repo